app/frontend/src/features/profile/profilePage/ProfilePage.tsx

```diff
@@ -87,7 +88,7 @@ export default function ProfilePage() {
                 <About {...{ user }} />
               </TabPanel>
               <TabPanel value="home">
-                <div>home</div>
+                <Home {...{ user }}></Home>
```

Comment:
```suggestion
                <Home user={user}}></Home>
```
Nitpick: Use the more explicit syntax to make it clear it's only one prop please :) The spread syntax is typically used for a "wildcard" of props where you want to forward them to child components.